torchgeo/datasets/bigearthnet.py

```diff
@@ -113,6 +113,11 @@ class BigEarthNet(NonGeoDataset):
     17. Inland waters
     18. Marine waters
 
+    The source for the above dataset classes, their respective ordering, and
+    43-to-19-class mappings can be found here:
+
+    * https://git.tu-berlin.de/rsim/BigEarthNet-S2_19-classes_models/-/blob/master/label_indices.json
```

Comment:
flake8 is failing because it doesn't like the long line. If you add `  # noqa: E501` at the end of the closing `"""` it should ignore this.